DockService.Infrastructure/Migrations/20180523104734_Trackfix01.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace DockService.Infrastructure.Migrations
+{
+    public partial class Trackfix01 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
+                name: "PK_Ships",
+                table: "Ships");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "DBKey",
+                table: "Ships",
+                nullable: false,
+                defaultValue: new Guid("00000000-0000-0000-0000-000000000000"));
```

Comment:
If you want to auto generate Guids, Add the following code in DbContext:

//protected override void OnModelCreating(ModelBuilder modelBuilder)
		{
			modelBuilder.Entity<EntityName>()
				.Property(x => x.Id)
				.ValueGeneratedOnAdd()
				.HasDefaultValueSql("NEWID()");
		}